src/debuginfo/unwind.rs

```diff
@@ -71,7 +71,11 @@ impl UnwindContext {
         }
     }
 
-    #[cfg(feature = "jit")]
+    #[cfg(all(feature = "jit", windows))]
+    pub(crate) unsafe fn register_jit(self, jit_module: &cranelift_jit::JITModule) {
+    }
```

Comment:
```suggestion
    pub(crate) unsafe fn register_jit(self, _jit_module: &cranelift_jit::JITModule) {}
```
To keep rustfmt and a rustc lint happy.